documentation/modules/exploit/linux/snmp/net_snmpd_rw_access.md

```diff
@@ -0,0 +1,87 @@
+## Vulnerable Application
+
+  This module uses SNMP extension MIBs to enable remote code execution on the Linux Net-SNMPD servers using the 
+  SNMP-EXTEND-MIB.
+
+## Verification Steps
+
+  1. Start `msfconsole`
+  2. Do: `use exploit/linux/snmp/net_snmpd_rw_access`
+  3. Do: `set rhost [IP]`
+  4. Do: `set community [SNMP Community]`
+  5. Do: `set version [SNMP Version]`
+  4. Configure the payload
+  5. Do: `run`
+  6. You should get a session
+
+
+## Example Run
```

Comment:
h00die, lord of documentation, will probably want you to label this section `Scenarios`